http-gateway/swagger.yaml

```diff
@@ -132,6 +132,34 @@ paths:
           $ref: '#/components/responses/deviceNotFound'
         500:
           $ref: '#/components/responses/error'
+    delete:
+      tags:
+        - Devices
+      summary: Delete a single resource
+      security:
+        - oauth2:
+          - plgd.devices
+      parameters:
+        - $ref: '#/components/parameters/deviceId'
+        - $ref: '#/components/parameters/resourcePath'
+      description: Delte a single resource
```

Comment:
Please be more explicit in descriptions, same as with comments in the go. You can take description from there.